Makefile

```diff
@@ -9,66 +9,73 @@
-	
-	tar xvf data/alov/frames/alov300++_frames.zip 
+
+	tar xvf data/alov/frames/alov300++_frames.zip
 
 	mv imagedata++ data/alov/frames/
 
-# Bounding Boxes # 
-data/alov/bb/alov300++GT_txtFiles.zip: 
+# Bounding Boxes #
+data/alov/bb/alov300++GT_txtFiles.zip:
 	curl http://isis-data.science.uva.nl/alov/alov300++GT_txtFiles.zip \
 		-o data/alov/bb/alov300++GT_txtFiles.zip
 
-	unzip data/alov/bb/alov300++GT_txtFiles.zip 
+	unzip data/alov/bb/alov300++GT_txtFiles.zip
 
 	mv alov300++_rectangleAnnotation_full/ data/alov/bb/bb
 
 alov: data/alov/frames/alov300++_frames.zip \
-	data/alov/bb/alov300++GT_txtFiles.zip 
+	data/alov/bb/alov300++GT_txtFiles.zip
 
 ##############
-# ILSVRC2014 # 
+# ILSVRC2014 #
 ##############
 
-# Training Bounding Boxes # 
-data/ILSVRC/bb/ILSVRC2014_DET_bbox_train.tgz: 
+# Training Bounding Boxes #
+work/.imagenet_training_bbox_sentinel:
 	curl http://image-net.org/image/ilsvrc2014/ILSVRC2014_DET_bbox_train.tgz \
 		-o data/ILSVRC/bb/ILSVRC2014_DET_bbox_train.tgz
-
 	tar xvf data/ILSVRC/bb/ILSVRC2014_DET_bbox_train.tgz
-
 	mv ILSVRC2014_DET_bbox_train/ data/ILSVRC/bb/training
+	rm data/ILSVRC/bb/ILSVRC2014_DET_bbox_train.tgz
+	touch work/.imagenet_training_bbox_sentinel
 
-# Validation Bounding Boxes # 
-data/ILSVRC/bb/ILSVRC2013_DET_bbox_val.tgz: 
-	curl http://image-net.org/image/ilsvrc2013/ILSVRC2013_DET_bbox_val.tgz \
-		-o data/ILSVRC/bb/ILSVRC2013_DET_bbox_val.tgz
 
-	tar xvf data/ILSVRC/bb/ILSVRC2013_DET_bbox_val.tgz
-
-	mv ILSVRC2013_DET_bbox_val/ data/ILSVRC/bb/validation
-	
-work/ILSVRC/parsed_training_bb.csv: 
+work/ILSVRC/parsed_training_bb.csv: work/.imagenet_training_bbox_sentinel
 	bash code/data_setup/parse_xml.sh data/ILSVRC/bb/training/ \
 		work/ILSVRC/parsed_training_bb.csv
 
-work/ILSVRC/parsed_validation_bb.csv: 
+work/ILSVRC/parsed_validation_bb.csv: work/.imagenet_validation_bbox_sentinel
 	bash code/data_setup/parse_xml.sh data/ILSVRC/bb/validation/ \
 				work/ILSVRC/parsed_validation_bb.csv
 
-ILSVRC: data/ILSVRC/bb/ILSVRC2014_DET_bbox_train.tgz \
-	data/ILSVRC/bb/ILSVRC2013_DET_bbox_val.tgz \
-	work/ILSVRC/parsed_training_bb.csv \
-	work/ILSVRC/parsed_validation_bb.csv 
 
-data: .folder_structure_sentinel ILSVRC 
+data/fall11_imagenet_urls.txt:
+	curl http://image-net.org/imagenet_data/urls/imagenet_fall11_urls.tgz \
+	  -o data/imagenet_fall11_urls.tgz
+	tar xvf data/imagenet_fall11_urls.tgz
+	rm data/imagenet_fall11_urls.tgz
+	mv fall11_urls.txt data/fall11_imagenet_urls.txt
+
+
+work/.imagenet_training_images_sentinel: code/data_setup/download_images.py data/fall11_imagenet_urls.txt
+	python code/data_setup/download_images.py "training"
+	touch work/.imagenet_training_images_sentinel
+
+work/.imagenet_validation_images_sentinel: code/data_setup/download_images.py data/fall11_imagenet_urls.txt
+	python code/data_setup/download_images.py "validation"
+	touch work/.imagenet_validation_images_sentinel
+
+
+imagenet_bb: work/ILSVRC/parsed_training_bb.csv work/ILSVRC/parsed_validation_bb.csv
```

Comment:
this replaces ILSVRC.  I find imagenet easier to remember.  If you think we're going to get more imagenet datasets later on and we want to disambiguate, let's think of a way to do that.